tck/features/uncategorized/FunctionsAcceptance.feature

```diff
@@ -121,24 +93,24 @@ Feature: FunctionsAcceptance
       | null             |
     And no side effects
 
-  Scenario: `exists()` with dynamic property lookup
-    Given an empty graph
-    And having executed:
-      """
-      CREATE (:Person {name: 'foo'}),
-             (:Person)
-      """
+  Scenario Outline: `exists()` with literal maps
```

Comment:
The scenario outline "`exists()` with dynamic property lookup" should be in "Map4 - Exists function" by commit https://github.com/RedisGraph/openCypher/commit/4ae580e8880575d63d0daec6114e4446edc4894b

It seems from https://github.com/RedisGraph/openCypher/commits/elemntary_graph_operations that you have rebase the our branch, however it still wants to duplicate this scenario back into uncategorized/FunctionsAcceptance. Please double check.

Reply:
removed